docs/content/docs/engines/spark.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Spark"
+weight: 3
+type: docs
+aliases:
+- /engines/spark.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Spark
+
+Table Store supports reading table store tables through Spark.
+
+## Version
+
+Table Store supports Spark 3+. It is highly recommended to use Spark 3+ version with many improvements.
+
+## Install
+
+{{< stable >}}
+Download [flink-table-store-spark-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-table-store-spark/{{< version >}}/flink-table-store-spark-{{< version >}}.jar).
+{{< /stable >}}
+{{< unstable >}}
+
+You are using an unreleased version of Table Store. The simplest way of building Table Store is by running:
+
+```bash
+mvn clean install -DskipTests
+```
+
+You can find the bundle jar in `"./flink-table-store-spark/target/flink-table-store-spark-{{< version >}}.jar"`.
+
+{{< /unstable >}}
+
+Copy Table Store Spark bundle jar to `spark/jars`.
+
+## Table Store Catalog
+
+The following command registers the Table Store's Spark catalog with the name `table_store`:
+
+```bash
+spark-sql --conf spark.sql.catalog.table_store=org.apache.flink.table.store.spark.SparkCatalog \
+    --conf spark.sql.catalog.table_store.warehouse=file:/tmp/warehouse
+```
+
+## Create Temporary View
+
+If you don't want to use Table Store Catalog. The CREATE TEMPORARY VIEW command
```

Comment:
`If you don't want to use Table Store Catalog.`  appears to be an incomplete sentence.
How about

```suggestion
Use the `CREATE TEMPORARY VIEW` command to create a Spark mapping table if you don't want to use Table Store Catalog.
```